### ISSUE TITLE
studiomodel not visible

### ISSUE THREAD

**Issue by MeaningO** [hzqst/MetaHookSv#509]
player studiomodel not visible (such as playermodel entity uncontrollable)
![20241112011624_1](https://github.com/user-attachments/assets/9dbe0619-3c8f-4d95-a99e-2ec5388c8016)
![20241112011623_1](https://github.com/user-attachments/assets/88063520-7cbb-435c-8935-1d36a4b08eb9)
these screenshots are made simultaneously
the glowing ones are entities created by non-glowing players(clones)

**Comment by hzqst** [comment]
are you running the latest version of Renderer.dll?

**Comment by MeaningO** [comment]
I run the one that came with the metahook itself

**Comment by hzqst** [comment]
use console command `mh_pluginlist` to check plugin versions.

**Comment by MeaningO** [comment]
|    6| v4|       Renderer_AVX2.dll|     2024-10-14T12:14:24|

**Comment by hzqst** [comment]
yes it is, so it's like the glowing entities may get invisible in certain case? or it's not just glowing ones but any other kRenderNormal entities?

**Comment by MeaningO** [comment]
all the playermodels, which are uncontrolled by players, are always invisible, it does not matter if they are glowing or not

**Comment by MeaningO** [comment]
these two glowing playermodels are "illusions" of the players (copying animation frame, position etc)

**Comment by hzqst** [comment]
Could it be the case that there were too many entities in the view? iirc engine 8684(pre-25th legacy version) supports up to 256 packet entities and any entitiy out this limit will be always invisible.

**Comment by MeaningO** [comment]
no, the screenshots are made with approximately 10 entities at the time
moreover, they're still invisible even if I'm the only one entity(alone at the server), creating the second one

**Comment by MeaningO** [comment]
by that I mean that it does not matter, if I'm the only one player, or there is full server, these models will always be invisible

**Comment by hzqst** [comment]
is there any minimal reproducible example? like a box bsp with 10 cycler entities?

**Comment by MeaningO** [comment]
no, i don't think so

**Comment by MeaningO** [comment]
upd: the "additive" flag does not render while on player (hats, for example)
it might be the reason

**Comment by hzqst** [comment]
> upd: the "additive" flag does not render while on player (hats, for example) it might be the reason

I wonder if the "uncontrollable player" models are `non-kRenderNormal`, or say they are like `kRenderTransAlpha` or `kRenderTransTexture`?

**Comment by hzqst** [comment]
Fixed in https://github.com/hzqst/MetaHookSv/releases/tag/v20250310b
